.github/CONTRIBUTING.md

```diff
@@ -125,8 +125,8 @@ $ sudo -u postgres psql -c "CREATE EXTENSION pg_trgm;" peertube_dev
 $ sudo -u postgres psql -c "CREATE EXTENSION unaccent;" peertube_dev
 ```
 
-Peertube also requires a running redis server, no special setup is needed for
-this.
+PeerTube also requires a running redis server, no special setup is needed for
+this (it's `sudo service redis-server start` on WSL 1 with Ubuntu).
```

Comment:
```suggestion
this.
```

Thank you @Chocobozzz! I will try to write something good over at [`./support/doc/dependencies.md`](https://github.com/Chocobozzz/PeerTube/blob/develop/support/doc/dependencies.md) documentation instead when I have everything working well under Windows/WSL. Removing this remark!